app/commands/user/reset_account.rb

```diff
@@ -33,6 +33,7 @@ def reset_mentoring!
       user.mentor_discussions.update_all(mentor_id: User::GHOST_USER_ID)
       user.mentor_discussion_posts.update_all(user_id: User::GHOST_USER_ID)
       user.mentor_testimonials.update_all(mentor_id: User::GHOST_USER_ID)
+      user.mentor_testimonials.update_all(mentor_id: User::GHOST_USER_ID)
```

Comment:
Isn't this the exact same line as the line above it?

Reply:
Dammed fat vim fingers. Yes. Thanks.